cupy/cuda/cupy_nccl.h

```diff
@@ -210,4 +222,14 @@ void ncclCommAbort(ncclComm_t comm) {
 }
 #endif
 
+#if (NCCL_VERSION_CODE < 2200)
```

Comment:
Would you move this stub before the definition of `_ncclBroadcast` function so that it can find the definition on an environment with the older NCCL? It's the cause of the CI failure.

Reply:
Thank you for your detailed review! will fix soon in that way.